### DISCUSSION TITLE
Why can't I get a spot light to work in shadowtest?

### DISCUSSION THREAD

**Discussion by jwwalker** [google/filament#3853] [Q&A] [answered]
I tried to change the `shadowtest` sample to use a spot light instead of a sun light, by changing the code that builds the light to this:

```c++
        LightManager::Builder(LightManager::Type::SPOT)
            .intensity(1.1e8)
            .direction({ 0.0, -1.0, 0.0 })
            .position({ 0.0, 2.0, -4.0 })
            .falloff(100.0)
            .spotLightCone( 1.0, 1.0 )
            .castShadows(ENABLE_SHADOWS)
            .build(*engine, app.light);
```

I can see that the monkey is lit from above, but there is no trace of shadow on the ground plane.  What am I missing?

![shadowtest screens shot](https://user-images.githubusercontent.com/730426/115967575-3e541d80-a4e8-11eb-9c50-dabb82e04351.png)

**Comment by romainguy** [comment, marked as answer]
The ground plane is a special invisible shadow receiver meant for AR and it only supports directional shadows.

**Reply by jwwalker** [reply]
OK, is there someplace in the documentation that would tell me that the ground plane only supports directional shadows?  For instance, the discussion of the `shadowMultiplier` attribute in the Filament Materials Guide doesn't say anything about that.

**Reply by bejado** [reply]
I'll update our documentation.
